CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.6-dev1
```

Comment:
Instead of a new heading, just change `0.2.6-dev0` to `0.2.6-dev1` and add your new bullet under there. Once we're ready to release, we'll change to `0.2.6` and the bullets will become the release notes.